<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="leFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>12</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="statusTip">
          <string>文件路径（&quot;Tab键&quot;或&quot;Shift键 + Tab键&quot;切换输入框，&quot;Space键&quot;点击，&quot;Esc键&quot;退出程序）</string>
         </property>
         <property name="whatsThis">
          <string>文件路径</string>
         </property>
         <property name="placeholderText">
          <string>File Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leKeyword">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="statusTip">
          <string>关键词，存在前边界时可留空</string>
         </property>
         <property name="whatsThis">
          <string>关键词</string>
         </property>
         <property name="placeholderText">
          <string>Keyword</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEdge">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="statusTip">
          <string>前边界，存在关键词时可留空</string>
         </property>
         <property name="whatsThis">
          <string>前边界</string>
         </property>
         <property name="placeholderText">
          <string>Edge A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEdge_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="statusTip">
          <string>后边界，默认为换行符</string>
         </property>
         <property name="whatsThis">
          <string>后边界</string>
         </property>
         <property name="placeholderText">
          <string>Edge B (\n)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_Simplify">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="statusTip">
          <string>是否简化内容</string>
         </property>
         <property name="whatsThis">
          <string>简化内容</string>
         </property>
         <property name="text">
          <string>Simplify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_KE">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="statusTip">
          <string>是否保留边界</string>
         </property>
         <property name="whatsThis">
          <string>保留边界</string>
         </property>
         <property name="text">
          <string>Keep Edge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="statusTip">
          <string>查找</string>
         </property>
         <property name="whatsThis">
          <string>查找</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
